templates/takeovers/_1710_takeover.html

```diff
@@ -0,0 +1,19 @@
+<section class="p-strip--image is-dark is-deep p-takeover--1710-release">
+  <div class="row">
+    <div class="u-equal-height u-vertically-center">
+      <div class="col-4 u-no-margin--bottom">
+        <h1 class="takeover-title">Ubuntu 17.10</h1>
+        <p class="intro-text">Brand new Ubuntu desktop!<br>
+        Same amazing experience!</p>
+        <p class="u-hidden u-visible--small">
+          <img class="laptop-image--small" src="{{ ASSET_SERVER_URL }}f9dcb391-Laptop.png?w=654" alt="" />
+        </p>
+        <p><a href="/download" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : '17.10 takeover', 'eventLabel' : 'Download 17.10 now', 'eventValue' : undefined });" class="p-button--neutral"><span>Download 17.10 now</span></a></p>
+        <p class="u-no-margin--bottom"><a href="/desktop/17.10" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Learn more', 'eventLabel' : 'Learn more about Ubuntu 17.10', 'eventValue' : undefined });" class="p-link--inverted">Learn more about Ubuntu 17.10&nbsp;›</a></p>
+      </div>
+      <div class="u-hidden--small col-8 u-vertically-center">
+        <img src="{{ ASSET_SERVER_URL }}f9dcb391-Laptop.png?w=654" alt="" />
```

Comment:
Can you add a descriptive alt tag here, please? Something like "Laptop running Ubuntu 17.10" or similar.